sig-scalability/charter.md

```diff
@@ -0,0 +1,110 @@
+# SIG Scalability Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README]
+and uses the Roles and Organization Management outlined in [sig-governance].
+
+[sig-governance]: https://github.com/kubernetes/community/blob/master/committee-steering/governance/sig-governance.md
+[Kubernetes Charter README]: https://github.com/kubernetes/community/blob/master/committee-steering/governance/README.md
+
+## Scope
+
+SIG Scalability primary responsibilities are to define scalability goals for
+Kubernetes, define, test and measure performance and scalability related
+Service Level Indicators (SLIs) and ensure that every Kubernetes release meets
+Service Level Objectives (SLOs) build on top of the above SLIs.
+
+We also coordinate and contribute to general system-wide scalability and
+performance improvements (that don't fall into the charter of another individual
+SIG) as well as provide consultations about any scalability and performance
+related aspects of Kubernetes.
+
+### In Scope
+
+#### Code, Binaries and Services:
+
+- Scalability and performance testing frameworks. Examples include:
+  - [Cluster loader](https://github.com/kubernetes/perf-tests/tree/master/clusterloader2)
+  - [Kubemark](https://github.com/kubernetes/kubernetes/tree/master/cmd/kubemark)
+- Scalability and performance tests:
+  - [Tests](https://github.com/kubernetes/kubernetes/blob/master/test/e2e/scalability/)
+
+#### Cross-cutting and Externally Facing Processes
+
+- Defining what does it mean that “Kubernetes scales”. This includes defining
+(or approving) individual performance SLIs/SLOs, ensuring they are all oriented
+on user experience and consistent with each other:
+  - [SLIs/SLOs](https://github.com/kubernetes/community/blob/master/sig-scalability/slos/slos.md)
+- Ensuring that each official Kubernetes release satisfies all scalability and
+performance related requirements, as stated in "Kubernetes scalability" definition.
+- Establishing and documenting best practises on how to design and/or implement
+Kubernetes features in scalable and performant way. Educating contributors and
+consulting individual designs/implementations to ensure that those are widely used.
+Example artifacts:
+  - [Scalability governance](https://github.com/kubernetes/community/blob/master/sig-scalability/governance)
+
+
+TODO: Figure out how to phrase finding bottlenecks, coordinating improvements
+and architectural changes, etc.
+
+### Out of scope
+
+- Improving performance/scalability of features falling into charters of
```

Comment:
Really? b/c we've really refactored the heck out of some features.

Reply:
Agree - though we don't have enough capacity to do that (it shouldn't be out missing).
Though in some cases we may help with that...